Rudder-Branch.podspec

```diff
@@ -2,6 +2,11 @@ require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
+branch_sdk_version = '~> 3.0.0'
+rudder_sdk_version = '~> 1.18'
+deployment_target = '12.0'
```

Comment:
Where are we using this ?

Reply:
do you intend to use it for `s.platform` as well? 

Otherwise, we can directly specify the value at s.ios.deployment_target

Reply:
I'm not sure, if we should use that value for s.platform or not. What are your thoughts?

Reply:
In general, when you support only a single platform, `s.platform` alone should be enough to declare the supported platform and the deployment_target of that platform. 

And `s.{platform}.deployment_target` is needed when you support multiple platforms and you want to specify the deployment target per platform level. 

In this case, usage of `s.platform` and `s.ios.deployment_target` is redundant and we can ignore `s.ios.deployment_target`. 

For more details refer [here](https://guides.cocoapods.org/syntax/podspec.html#group_platform)

Reply:
Thanks. Next, I'll remove the `s.ios.deployment_target` and only keep `s.platform`. And also remove `deployment_target` and hard code the value.